Disc/Networking/Authorization/GetAccessToken.swift

```diff
@@ -14,16 +14,26 @@ private struct GetAccessTokenRequest: Request {
         body = createRequestParameters(clientId: clientId, clientSecret: clientSecret, code: code)
     }
     
-    init(clientId: String, scopes: [Scope] = [], provider: Provider, token: String, secret: String?) {
+    init(scopes: [Scope] = [], provider: Provider, token: String, secret: String?) {
         url = GetAccessTokenRequest.authLoginUrl
         body = createRequestParameters(scopes: scopes, provider: provider, token: token, secret: secret)
     }
     
-    init(clientId: String, scopes: [Scope] = [], provider: Provider, code: String, redirectUri: String) {
+    init(scopes: [Scope] = [], provider: Provider, code: String, redirectUri: String) {
         url = GetAccessTokenRequest.authLoginUrl
         body = createRequestParameters(scopes: scopes, provider: provider, code: code, redirectUri: redirectUri)
     }
     
+    init(clientId: String, scopes: [Scope] = [], provider: Provider, token: String, secret: String?) {
+        url = GetAccessTokenRequest.authLoginUrl
+        body = createRequestParameters(clientId: clientId, scopes: scopes, provider: provider, token: token, secret: secret)
+    }
+    
+    init(clientId: String, scopes: [Scope] = [], provider: Provider, code: String, redirectUri: String) {
+        url = GetAccessTokenRequest.authLoginUrl
+        body = createRequestParameters(clientId: clientId, scopes: scopes, provider: provider, code: code, redirectUri: redirectUri)
+    }
+    
```

Comment:
I think updating the tests as mentioned will make it more obvious that these constructors are never used.